src/VirtoCommerce.ContentModule.Web/Scripts/blades/pages/page-detail.js

```diff
@@ -117,10 +117,10 @@ angular.module('virtoCommerce.contentModule')
                         $scope.bladeClose();
                         blade.parentBlade.refresh();
                     },
-                    function (error) { bladeNavigationService.setError('Error ' + error.status, blade); });
+                        function (error) { bladeNavigationService.setError('Error ' + error.status, blade); });
```

Comment:
Probably you can delete this line, as it should be handled by the global interceptor.

Reply:
removed